scripts/coverage.svg

```diff
@@ -1 +1,7 @@
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="94" height="20"><linearGradient id="b" x2="0" y2="100%"><stop offset="0" stop-color="#bbb" stop-opacity=".1"/><stop offset="1" stop-opacity=".1"/></linearGradient><clipPath id="a"><rect width="94" height="20" rx="3" fill="#fff"/></clipPath><g clip-path="url(#a)"><path fill="#555" d="M0 0h59v20H0z"/><path fill="#dfb317" d="M59 0h35v20H59z"/><path fill="url(#b)" d="M0 0h94v20H0z"/></g><g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="11"><text x="29.5" y="15" fill="#010101" fill-opacity=".3">coverage</text><text x="29.5" y="14">coverage</text><text x="75.5" y="15" fill="#010101" fill-opacity=".3">87%</text><text x="75.5" y="14">87%</text></g></svg>
\ No newline at end of file
+<html>

```

Comment:
@eduardoboucas this is obviously incorrect, but my preference for this file would be to remove and ignore it in the repo

Reply:
Agree!